pkgs/development/python-modules/qbittorrent-api/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, six
+, urllib3
+, packaging
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "qbittorrent-api";
+  version = "2023.7.52";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-RHOupNo0jteUpxcxAojOfnBGGBt293j0OCHeKEritpQ=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    six
+    urllib3
+    packaging
+  ];
+
+  buildInputs = [
+    setuptools
+  ];
```

```suggestion
  nativeBuildInputs = [
    setuptools
    wheel
  ];
```
We will have to add  `wheel` to `nativeBuildInputs`.
see https://github.com/NixOS/nixpkgs/pull/245509